lnbits/settings.py

```diff
@@ -1,92 +1,298 @@
+        except:
+            logger.error("could not import module lnbits.extensions.admin database")
+            raise
+
+        async with ext_db.connect() as db:
+            try:
+                row = await db.fetchone("SELECT * FROM admin.settings")
+                if not row or len(row) == 0:
+                    logger.warning(
+                        "admin.settings empty. inserting new settings and creating admin account"
+                    )
+
+                    from lnbits.core.crud import create_account
+
+                    account = await create_account()
+                    settings.lnbits_admin_users.insert(0, account.id)
+                    keys = []
+                    values = ""
+                    for key, value in settings.dict(exclude_none=True).items():
+                        keys.append(key)
+                        if type(value) == list:
+                            joined = ",".join(value)
+                            values += f"'{joined}'"
+                        if type(value) == int or type(value) == float:
+                            values += str(value)
+                        if type(value) == bool:
+                            values += "true" if value else "false"
+                        if type(value) == str:
+                            value = value.replace("'", "")
+                            values += f"'{value}'"
+                        values += ","
+                    q = ", ".join(keys)
+                    v = values.rstrip(",")
+                    sql = f"INSERT INTO admin.settings ({q}) VALUES ({v})"
+                    await db.execute(sql)
+                    logger.warning(
+                        "initialized admin.settings from enviroment variables."
+                    )
+
+                    row = await db.fetchone("SELECT * FROM admin.settings")
+                    assert row, "Newly updated settings couldn't be retrieved"
+
+                admin = Settings(**row)
+
+                for key, value in admin.dict(exclude_none=True).items():
+                    if not key in read_only_variables:
+                        try:
+                            setattr(settings, key, value)
+                        except:
+                            logger.error(
+                                f"error overriding setting: {key}, value: {value}"
+                            )
+
+                logger.debug(f"Admin settings:")
+                for key, value in settings.dict(exclude_none=True).items():
+                    logger.debug(f"{key}: {value}")
+
+                http = "https" if settings.lnbits_force_https else "http"
+                user = settings.lnbits_admin_users[0]
+
+                admin_url = (
+                    f"{http}://{settings.host}:{settings.port}/wallet?usr={user}"
+                )
+                logger.warning(f"✔️ Access admin user account at: {admin_url}")
```

Comment:
This is not a warning. Use `logger.info`.

Reply:
i did it for better visibility, because the admin url is quite important, should i use success?

Reply:
9b37c68b